Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -40,6 +41,9 @@ module Livecheck
     # and prints the results.
     # @return [nil]
     def livecheck_formulae(formulae_to_check, args)
+      # If args is a hash, convert it to an OpenStruct so that options can be accessed using methods
```

Comment:
Would be nicer to use keyword arguments here as otherwise this'll break on Ruby 2.7.

Reply:
Would this be okay?
```ruby
def livecheck_formulae(formulae_to_check, args:)
```
`args` isn't always going to be a hash, it could even be parsed arguments from the parser. Calling the method using `livecheck_formulae(formulae_to_check, args: args)` where `args` is either a hash or the parsed args would prevent the warning/incompatibility, wouldn't it?

Wasn't sure if you meant this or something like `def livecheck_formulae(formulae_to_check, quiet: false, verbose: false, ...)`. Let me know what you think!

Reply:
> Wasn't sure if you meant this or something like `def livecheck_formulae(formulae_to_check, quiet: false, verbose: false, ...)`

Yeh, this ideally 👍🏻

Reply:
I'm moving this suggestion to another PR that addresses the suggestion on abstracting the `puts` out of `livecheck_formulae` and the audit, as it's not required in this PR.